<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MT-PF185WZ2" failures="3" tests="4" name="Test" time="17.645" errors="0" timestamp="2021-09-20T13:13:40 IST">
  <testcase classname="REST.restRunner" name="ApiTest" time="17.338"/>
  <testcase classname="Soap.testRunner2" name="SingleTest" time="0.172">
    <failure type="java.lang.AssertionError" message="expected [FAILED] but found [FINISHED]">
      <![CDATA[java.lang.AssertionError: expected [FAILED] but found [FINISHED]
at Soap.testRunner2.SingleTest(testRunner2.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- SingleTest -->
  <testcase classname="REST.restRunner2" name="ApiTest2" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [FAILED] but found [FINISHED]">
      <![CDATA[java.lang.AssertionError: expected [FAILED] but found [FINISHED]
at REST.restRunner2.ApiTest2(restRunner2.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- ApiTest2 -->
  <testcase classname="Soap.testRunner" name="SoapTest" time="0.026">
    <failure type="java.lang.AssertionError" message="expected [FAILED] but found [FINISHED]">
      <![CDATA[java.lang.AssertionError: expected [FAILED] but found [FINISHED]
at Soap.testRunner.SoapTest(testRunner.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- SoapTest -->
</testsuite> <!-- Test -->
